packages/cactus-plugin-ledger-connector-daml/tsconfig.json

```diff
@@ -0,0 +1,33 @@
+{
+    "extends": "../../tsconfig.base.json",
+    "compilerOptions": {
+      "composite": true,
+      "outDir": "./dist/lib",
+      "declarationDir": "./dist/lib",
+      "resolveJsonModule": true,
+      "rootDir": "./src",
+      "tsBuildInfoFile": "../../.build-cache/cactus-plugin-persistence-daml.tsbuildinfo"
+    },
+    "include": ["./src", "./src/**/*.json", "./src/**/*.sql"],
+    "references": [
+      {
+        "path": "../cactus-core/tsconfig.json"
+      },
+      {
+        "path": "../cactus-common/tsconfig.json"
+      },
+      {
+        "path": "../cactus-core-api/tsconfig.json"
+      },
+      {
+        "path": "../cactus-test-tooling/tsconfig.json"
+      },
+      {
+        "path": "../cactus-plugin-keychain-memory/tsconfig.json"
+      },
+      {
+        "path": "../cactus-cmd-api-server/tsconfig.json"
+      }
+    ]
```

Comment:
@raynatopedrajeta Connectors can't depend on the API server package because it will create circular dependencies (the API server has to depend on all plugins first)
Please eliminate this dependency.

Tests that verify the plugin's integration with the API server should be in a separate test package (if that's the reason for this)